packages/plugin-apex/src/commands/force/apex/test/run.ts

```diff
@@ -163,6 +163,7 @@ export default class Run extends SfdxCommand {
         this.flags.tests,
         this.flags.classnames
       );
+      payload.skipCodeCoverage = this.flags.codecoverage ? false : true;
```

Comment:
Wouldn't these be better in the TestService buildXXX methods?

Reply:
That was exactly the conversation we had yesterday on my team! I felt the same, but it's a much more intrusive change to make there, and no one on the team had a strong objections about putting it in the safer spot here. @rcoringrato-sfdc let me know if this bumps a little more now that the change is in context.